### ISSUE TITLE
Do we still need the marathon_gui?

### ISSUE THREAD

**Issue by cdondrup** [strands-project/strands_ui#45]
This is, if I'm not mistaken, the gui we ran last year. Is this still a useful package?
The reason I'm asking is because it depends on autonomous patrolling:

```
marathon_touch_gui/package.xml:  <build_depend>ap_msgs</build_depend>  
marathon_touch_gui/package.xml:  <run_depend>ap_msgs</run_depend>    
marathon_touch_gui/index.html:        serviceType : 'ap_msgs/PauseResumePatroller'
marathon_touch_gui/CMakeLists.txt:  ap_msgs
marathon_touch_gui/js/marathon.js:        serviceType : 'ap_msgs/GetLatestEpisodeName'
marathon_touch_gui/js/marathon.js:        serviceType : 'ap_msgs/GetEpisodeSummary'
```

Since I would like to release this repo soon and I'm not sure what's going to happen to the autonomous_patrolling one, I would love to get rid of that dependency.

Maybe @hawesie could clarify if that package is still useful or could go to some kind of attic?

**Comment by hawesie** [comment]
I think this can be removed. We will need something like it, but changes will be necessary (that we can't do today). Maybe stick it in another branch?

**Comment by cdondrup** [comment]
Will create an attic branch until it has been decided what to do with it. Thanks!

**Comment by cdondrup** [comment]
Closed via #46
